vector/src/main/java/im/vector/app/features/home/room/detail/TimelineViewModel.kt

```diff
@@ -1231,6 +1233,28 @@ class TimelineViewModel @AssistedInject constructor(
         }
     }
 
+    private fun observeLocalRoomSummary() {
+        if (room != null && RoomLocalEcho.isLocalEchoId(room.roomId)) {
+            room.flow().liveLocalRoomSummary()
+                    .unwrap()
+                    .map { it.creationState }
+                    .distinctUntilChanged()
+                    .onEach { creationState ->
+                        when (creationState) {
+                            LocalRoomCreationState.NOT_CREATED -> Unit
+                            LocalRoomCreationState.CREATING ->
+                                _viewEvents.post(RoomDetailViewEvents.ShowWaitingView(stringProvider.getString(R.string.creating_direct_room)))
+                            LocalRoomCreationState.FAILURE -> {
+                                _viewEvents.post(RoomDetailViewEvents.HideWaitingView)
```

Comment:
Can we provide some detail of the failure to the user? (maybe not blocking).
Also, not tested, but maybe try to create a DM in airplane mode to ensure that there is no crash.

Reply:
I tried, sometimes the loading wheel is shown before being hidden, sometimes not, depending on the event position in the events queue. There is no crash

Reply:
In case of error (network or server), we just have the same feedback as for a normal message. We should probably sync with the product to show more details:
<img width="522" alt="image" src="https://user-images.githubusercontent.com/11990514/191048352-10368897-dc33-4dd3-b45c-869c05898417.png">